mecanum_drive_controller/test/test_mecanum_drive_controller_preceeding.cpp

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) 2023, Stogl Robotics Consulting UG (haftungsbeschränkt)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "test_mecanum_drive_controller.hpp"
+
+#include <limits>
+#include <memory>
+#include <string>
+#include <utility>
+#include <vector>
+
+using mecanum_drive_controller::NR_CMD_ITFS;
+using mecanum_drive_controller::NR_REF_ITFS;
+using mecanum_drive_controller::NR_STATE_ITFS;
+
+class MecanumDriveControllerTest
+: public MecanumDriveControllerFixture<TestableMecanumDriveController>
+{
+};
+
+TEST_F(MecanumDriveControllerTest, when_controller_is_configured_expect_all_parameters_set)
+{
+  SetUpController();
```

Comment:
Setting up the controller at the beginning will cause the test to fail later because the controller parameter is already configured in the function, according to my test, moving it to line 46, the test works fine.

Reply:
It looks like I opened a review by mistake, feel free to delete the review I submitted if needed!